README.md

```diff
@@ -140,7 +142,8 @@ accelerate launch scripts/finetune.py examples/openllama-3b/lora.yml \
   git clone https://github.com/OpenAccess-AI-Collective/axolotl
   cd axolotl
 
-  pip3 install -e .
+  pip3 install packaging
+  pip3 install -e .[flash-attn]
   pip3 install protobuf==3.20.3
   pip3 install -U --ignore-installed requests Pillow psutil scipy
```

Comment:
You can merge these other installs together.

Reply:
the problem is that if packaging isn't installed first, trying to install at the same time fails for flash-attn.